components/TProfile.vue

```diff
@@ -308,6 +312,19 @@ export default {
       default: () => ({}),
     },
   },
+  data() {
+    return {
+      showPopup: false,
+    }
+  },
+  mounted() {
+    // show tryPremium modal
+    const query = $nuxt.$route.query
```

Comment:
solution for: error  '$nuxt' is not defined

```suggestion
    const query = this.$route.query
```

Reply:
I had error in diferent file, so this does not fix anything, but I added this implementation